tests/modules/nf-core/mitohifi/findmitoreference/main.nf

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { MITOHIFI_FINDMITOREFERENCE } from '../../../../../modules/nf-core/mitohifi/findmitoreference/main.nf'
+
+workflow test_mitohifi_findmitoreference {
+    
+    species = "Homo sapiens"
+    email = "verena.kutschera@scilifelab.se"
```

Comment:
I wouldn't put your personal e-mail here, unless you want to receive constant e-mails when someone reruns the tests from this module :)

Reply:
Yes, good point... ;-)

Reply:
Do you have a suggestion for an email address I could enter instead? Like some kind of dummy email address?

Reply:
Go wild ;) but make sure it's not taken already so you won't spam anyone